source/slang/hlsl.meta.slang

```diff
@@ -812,99 +816,132 @@ float2 GetRenderTargetSamplePosition(int Index);
 __target_intrinsic(glsl, "groupMemoryBarrier")
 void GroupMemoryBarrier();
 
+
 __target_intrinsic(glsl, "groupMemoryBarrier(), barrier()")
+// TODO(JS): I don't think __syncwarp can be used because a 'group' could be multiple warps
+__target_intrinsic(cuda, "__syncthreads()")
```

Comment:
Correct. I don't think there is a way in the current HLSL programming model to do warp-level or sub-group synchronization.